### PR TITLE
Carolyn/create log sign in

### PR DESCRIPTION
**Pull request by carolynzhang18**
## Notion task link
https://www.notion.so/uwblueprintexecs/Create-Log-On-Sign-In-0364fbd4600f43628cc879bd84b196d2

Create Log On Sign-In (https://www.notion.so/uwblueprintexecs/2792c3504d744d2b90ff967c32d00961?v=d33629815ecf477ab4e1ff4e33774c32)

## Implementation description
* Call create_sign_in_log upon log in (auth_routes.py)
* Add env variable CREATE_SIGN_IN_LOG to control sign in logging

## Steps to test
1. Change env variable CREATE_SIGN_IN_LOG=True.
2. Log in.
3. Check sign_in_log table for new sign-in record.

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR
- [x] If I have made API changes, I have updated the [REST API Docs](https://www.notion.so/uwblueprintexecs/REST-Endpoints-05ce60312bb943439dfda42bb1318536)
- [x] IF I have made changes to the db/models, I have updated the [Data Models Page](https://www.notion.so/uwblueprintexecs/Data-Models-760f8aa06b244eb0842c079ad77987b0)
- [x] I have updated other Docs as needed

### CODE REVIEW

File: backend/app/services/implementations/sign_in_logs_service.py [hunk 1]
```diff
@@ -24,9 +25,12 @@ def __init__(self, logger):
     def create_sign_in_log(self, user_id):
         sign_in = {"id": user_id, "time": datetime.now()}
         try:
-            new_sign_in = SignInLogs(**sign_in)
-            db.session.add(new_sign_in)
-            db.session.commit()
+            print("!!!!LADBLNALINPIWGAEN")
```
Comment (braydonwang) [inline backend/app/services/implementations/sign_in_logs_service.py]: Lol are these print statements supposed to be here or were they for debugging

Comment (connor-bechthold) [reply backend/app/services/implementations/sign_in_logs_service.py]: +1 this should be deleted (`!!!!LADBLNALINPIWGAEN` is an interesting choice?)

File: backend/app/rest/auth_routes.py [hunk 2]
```diff
@@ -55,6 +55,7 @@ def login():
             auth_dto = auth_service.generate_token(
                 request.json["email"], request.json["password"]
             )
+
```
Comment (connor-bechthold) [inline backend/app/rest/auth_routes.py]: Remove this extra line (have you ran the linter?)

File: backend/app/rest/sign_in_logs_routes.py [hunk 3]
```diff
@@ -9,7 +10,7 @@
 
 user_service = UserService(current_app.logger)
 sign_in_logs_service = SignInLogService(current_app.logger)
-blueprint = Blueprint("logs", __name__, url_prefix="/logs")
+blueprint = Blueprint("sign_in_logs", __name__, url_prefix="/sign_in_logs")
```
Comment (connor-bechthold) [inline backend/app/rest/sign_in_logs_routes.py]: Best practice for REST URLs is to use hyphens (i.e. `sign-in-logs`)

File: backend/app/rest/sign_in_logs_routes.py [hunk 4]
```diff
@@ -1,3 +1,4 @@
+import os
```
Comment (connor-bechthold) [inline backend/app/rest/sign_in_logs_routes.py]: This can be removed

File: backend/app/services/implementations/sign_in_logs_service.py [hunk 5]
```diff
@@ -24,9 +25,12 @@ def __init__(self, logger):
...
+            print(os.getenv("CREATE_SIGN_IN_LOG"))
```
Comment (connor-bechthold) [inline backend/app/services/implementations/sign_in_logs_service.py]: This print statement can be removed

**Review by connor-bechthold — CHANGES_REQUESTED** [review]
GREAT JOB FOR YOUR FIRST PR!!!!!!!!! left a couple comments to address but overall functionality LGTM 👨‍🍳

File: backend/app/services/implementations/sign_in_logs_service.py [hunk 6]
```diff
@@ -24,9 +25,12 @@ def __init__(self, logger):
...
+            if (os.getenv("CREATE_SIGN_IN_LOG")):
```
Comment (connor-bechthold) [inline backend/app/services/implementations/sign_in_logs_service.py]: So by default, environment variables are interpreted as strings, which are truthy. This means that this function will either return `"True"` or `"False"`, which will both be interpreted as `True`. So, you need to do something like this:

`os.getenv("CREATE_SIGN_IN_LOG") == "True"`

File: backend/app/services/implementations/sign_in_logs_service.py (hunk 1 above)
Comment (carolynzhang18) [reply backend/app/services/implementations/sign_in_logs_service.py]: Oops missed that one... Thank you!

**Review by connor-bechthold — APPROVED** [review]
LGTM!!! Congrats on your first PR 🧨 🧨 🧨 

Remember to link this in the releases document

**Review by braydonwang — APPROVED** [review]
LGTM!

**Pull request merged**